app/controllers/catalog_controller.rb

```diff
@@ -68,7 +68,8 @@ def self.year_published_field
     config.default_solr_params = {
       qt: "search",
       rows: 10,
-      qf: "title_tesim description_tesim creator_tesim keyword_tesim all_text_timv"
+      qf: IiifPrint.config.metadata_fields.keys.map { |attribute| "#{attribute}_tesim" }
```

Comment:
Q in engineering channel to see if there's a better approach: https://assaydepot.slack.com/archives/G0313NK5NMA/p1681945717935149

Otherwise, this solution was inspired by work done in LV: https://gitlab.com/notch8/louisville-hyku/-/merge_requests/167/diffs